virtdata-lib-curves4/src/main/java/io/nosqlbench/virtdata/library/curves4/continuous/int_double/ConstantContinuous.java

```diff
@@ -32,6 +32,6 @@
 @Categories({Category.distributions})
 public class ConstantContinuous extends IntToDoubleContinuousCurve {
     public ConstantContinuous(double value, String... mods) {
-        super(new ConstantContinuousDistribution(value), mods);
```

Comment:
I can't find the docs on ConstantContinuousDistribution anymore:
[](https://commons.apache.org/proper/commons-statistics/commons-statistics-distribution/apidocs/org/apache/commons/statistics/distribution/ConstantContinuousDistribution.html)
Are we sure an EmpiricalDistribution will fullfill the same role?